lib/rubocop/cop/style/send.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for the use of the send method.
+      class Send < Cop
+        MSG = 'Prefer __send__ or public_send to send,
+               as send may overlap with existing methods.'
+
+        def on_send(node)
+          return if command?(:__send__, node) || command?(:public_send, node)
```

Comment:
This code is wrong (try it on code that has any method invocations in it), as you should actually be looking for usages of `send`. Using `command?` is a bad idea, as `send` can invoked without arguments.